examples/vision/mixUp.py

```diff
@@ -0,0 +1,219 @@
+"""
+Title: mixup augmentation for image classification
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2020/03/06
+Last modified: 2020/03/06
+Description: How to implement mixup to augment images for image classification problems.
```

Comment:
Suggestion: refactoring

```suggestion
Description: Applying the mixup technique for data augmentation to aid with image classification.
```

(Inspiration: https://keras.io/examples/vision/image_classification_from_scratch/)